src/psyclone/tests/parse/algorithm_test.py

```diff
@@ -274,7 +302,7 @@ def test_getkernel_isarg(parser, content):
 @pytest.mark.parametrize('content',
                          ["- arg", "1.0 * arg", "(1.0 * arg)",
                           "1.0 * (1.0 * arg)", "arg1*arg2"])
-def test_getkernel_noexpr(parser, content):
+def test_getkernel_noexpr(content):
     '''Test that the get_kernel function recognises an expression
     containing a variable and raises an exception (as this is not
     currently supported).
```

Comment:
pylint complains about useless suppression of unused-argument.

Reply:
Fixed complaints.